src/v/bytes/iobuf_parser.h

```diff
@@ -57,9 +57,14 @@ class iobuf_parser_base {
         return {val, length_size};
     }
 
-    ss::sstring read_string(size_t len) {
+    ss::sstring read_string_unsafe(size_t len) {
         ss::sstring str = ss::uninitialized_string(len);
         _in.consume_to(str.size(), str.begin());
+        return str;
+    }
+
```

Comment:
with 3 functions named `read_string_unsafe`, `read_string` and `read_string_safe` it'd be nice if all the 3 were documented